### ISSUE TITLE
VBA IA Request from Sitewide-Facilities 

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#64933]
## About your team

- Team name: Sitewide Facilities
- OCTO-DE product owner: Michelle Middaugh
- Product manager: Jane Xiong
- Designer(s): Jordan Wood and Dave Pickett
- Accessibility specialist: Laura Flannery
- Content writer: fields editors 
- Slack channel: #sitewide-facilities
 
### Describe what you need in a few sentences:
- We're launching new modernized pages for VBA Regional offices and need IA support. 

### Which are you doing?
- [x] **Launching a new** page, tool, form, or app

### Types of Support
What areas does your team need support in? Check all that apply.
#### IA
- [x] Page placement
- [x] URLs
- [x] Breadcrumbs and Left Nav placement
- [x] Redirects

## Timeframe
_We guide and work alongside teams across nearly every OCTO digital product. We also work with partners across VA to lead content migration and manage all unauthenticated content on VA.gov. And we're always working to expand the VA.gov content style guide and our IA and accessibility documentation. We need to prioritize all intake requests based on our overall workload and VA and OCTO priorities._

_Tell us about your product's timeline. And we'll work with you to meet timeline needs as best we can._

### Have you worked with CAIA before? 
  - [x] Yes
  - [ ] No

### What's your team’s next step and its timing?

- **Our next steps are** CMS content and FE builds
- **Provide date if available**: in progress now

#### Timing for your next step: 
  - [x] Next sprint - incorporating Veteran research into design
  - [x] This quarter - CMS content and FE builds, followed by editors entering content for the MVP sites

### Is this timing related to a specific event or Congressionally mandated deadline?
  - [x] Yes - launch
  - [ ] No

### If you're conducting research, when is that starting? 
**Provide date if available:** 
  - [x] Next week - Synthesis begins 

### Will you release this new product incrementally (for example, release to 25% of users to start)? 
 - the roll-out will begin with 7 Regional offices but these pages will be available to 100% of users. The react widget isn't  relevant to this work
  
### When do you expect to launch your product to 100% of users?
  
*Please provide an estimated date so our team can create other relevant tickets.*
- **We expect to launch our product to 100% of users by** 1Q2024
  
### Will this work be going through the Collaboration Cycle?
- [x] Yes 
- [ ] No

### If no ...
- Skip to the _supporting artifacts_ section, below.
 
#### Please provide the link to your Collaboration Cycle ticket: 
- **Ticket (number)**: Collaboration Cycle for Sitewide Facilities VBA Regional Offices #46029
- **Collab. Cycle Ticket Opened (date)**: 08/22/2022
 
#### Please check all of the phases in the Collaboration Cycle you have completed: 
- [x] Design Intent
- [x] Midpoint Review
- [ ] Staging
 
##### Design Intent
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [x] Already completed, no assistance needed
- [ ] Unknown at this time
##### Midpoint
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [x] Already completed, no assistance needed
- [ ] Unknown at this time
##### Staging
- [ ] Accessibility
- [x] Content
- [x] IA
- [ ] Translation
- [ ] Already completed, no assistance needed
- [ ] Unknown at this time
  
## Supporting artifacts
*Please provide links to supporting artifacts as available.*
- [x] [Product Outline](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/facilities/regional-offices)
- [x] [Finalized design prototype or mockup](https://www.sketch.com/s/891d33ae-152d-471c-ab5d-9aedf89cf6ff)
   - [VBA RO page](https://www.sketch.com/s/891d33ae-152d-471c-ab5d-9aedf89cf6ff/p/5712745E-0E95-47D2-9AAE-33F08EA3819E/canvas) mostly follows the patterns established for Vet Centers
   - [VBA RO page annotated](https://www.sketch.com/s/891d33ae-152d-471c-ab5d-9aedf89cf6ff) highlights areas where we are iterating on a pattern
   - [Facilities Services](https://www.sketch.com/s/891d33ae-152d-471c-ab5d-9aedf89cf6ff/p/9BAF00BD-1CA9-4FCF-A14B-E2B3D31A18E6/canvas) shows the design for an expanded facility service accordion (Veteran benefits, Family member and caregiver benefits, Service member benefits, Other benefits). VBA facility services are based primarily on the current VAMC design.
   -  [Facilities Services annotated](https://www.sketch.com/s/891d33ae-152d-471c-ab5d-9aedf89cf6ff/p/578C3207-C5A3-4303-AB18-4AFC27AC54D2/canvas) highlights iterations and additions from VAMC to VBA.
- [x] **Content source of truth**: This is a Drupal CMS product. Most content will be entered and maintained by local VA staff. Content will also include basic facility data (not editable in Drupal) and centralized content (created by editors from VBA Strategic Engagement for consistency across all sites and not locally editable). 
- [x] [**Research plan**](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/facilities/regional-offices/research/2023-06-veteran-facing/research-plan.md) 
   - Synthesis and findings pending
- **Any epics/issues that may be helpful**:
     - New proposed VBA terms for Service Taxonomy + Content Review from Facilities  #58511
     - Super Initiative - VBA Regional Offices modernization #13029
     - Initiative - VBA Regional Offices: Design & Develop MVP #10514
- please see additional context/history in comments  
- **Link to testing accounts** (please do not paste them in this ticket): _not relevant - these pages live in the  unauthenticated experience_

### Have you already worked with someone from OCTO leadership?
- [x] Danielle
- [ ] Beth
- [x] Mikki
- [ ] Martha

### How do you plan to work with us on this request?
- [ ] I’ll take this to Beth or Danielle’s office hours.
- [ ] I’d like to schedule time to talk about this.
- [x] Let’s work async and meet as needed.
- [ ] Schedule a kick-off call with your team and CAIA for your project **(Recommended)**

 
### Accessibility Help in Slack
The [#accessibility-help channel](https://dsva.slack.com/archives/C8E985R32) in the DSVA Slack, is also available to you as a resource. Everyone is welcome to ask questions or get help from our a11y (accessibility) specialists. Tag `@Terry Nichols` to get a11y help asap.

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag `@Terry Nichols`.**

**Comment by kristinoletmuskat** [comment]
This ticket seems to be a duplicate of [64934](https://github.com/department-of-veterans-affairs/va.gov-team/issues/64934), which has more comments from the stakeholder, so I'm going to close it.
